index.js

```diff
@@ -104,6 +105,46 @@ export async function registerSpace (email) {
   console.log(`⁂ space registered to ${email}`)
 }
 
+/**
+ * @param {string} proofPath
+ */
+export async function addSpace (proofPath) {
+  const client = await getClient()
+  const reader = await CarReader.fromIterable(fs.createReadStream(proofPath))
```

Comment:
verify proofPath exists first? catch error from CarReader if proof is invalid?

It looks better if we can console.error a warning rather than throwing an error where possible.

Reply:
https://github.com/web3-storage/w3cli/issues/15